src/test/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanelTest.kt

```diff
@@ -0,0 +1,148 @@
+package io.snyk.plugin.ui.toolwindow
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.util.registry.Registry
+import com.intellij.testFramework.LightPlatform4TestCase
+import com.intellij.testFramework.replaceService
+import com.intellij.ui.OnePixelSplitter
+import com.intellij.ui.treeStructure.Tree
+import io.mockk.every
+import io.mockk.justRun
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.unmockkAll
+import io.mockk.unmockkStatic
+import io.mockk.verify
+import io.snyk.plugin.services.SnykAnalyticsService
+import io.snyk.plugin.services.SnykApiService
+import io.snyk.plugin.services.SnykApplicationSettingsStateService
+import io.snyk.plugin.services.SnykTaskQueueService
+import org.junit.Test
+import snyk.UIComponentFinder
+import snyk.amplitude.AmplitudeExperimentService
+import java.awt.Container
+
+class SnykToolWindowPanelTest : LightPlatform4TestCase() {
```

Comment:
uasge of LightPlatform4TestCase - then it must be under IntegTest

Reply:
I would like to discuss that - as not every LightPlatform4TestCase is automatically an integration test :)

Reply:
Original separation agreed with @pavel-snyk was: fast Unit tests without any internal dependencies (like JB test framework) are under `test`
and everything else goes to `IntegTest`
We probably need to discuss that again. But till then I would follow that rule for consistency.

Reply:
changed it for now, but yeah, I would want to discuss it regarding semantics.